the-blue-alliance-ios/Core Data/Team/Team.swift

```diff
@@ -61,6 +47,7 @@ extension Team: Locatable, Managed {
             team.postalCode = model.postalCode
             team.rookieYear = Int16(model.rookieYear)
             team.stateProv = model.stateProv
+            // TODO: Store teamNumber as Int32/64 (64 is fine I think)
```

Comment:
Make sure this doesn't fall through the cracks

I don't believe we support any 32bit devices, and the Watch is safe to use 64bit data types on